instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -1839,3 +1839,50 @@ def test_custom_header_not_present_in_non_recording_span(self):
         self.assertEqual(200, resp.status_code)
         span_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(span_list), 0)
+
+
+class TestFastAPIRenderPathParameters(TestBaseFastAPI):
+
+    def test_render_path_parameters(self):
+        """Test that path parameters are rendered correctly in spans."""
+
+        # Create a FastAPI app with a path parameter
+        app = fastapi.FastAPI()
+
+        @app.get("/user/{username}")
+        async def read_user(username: str):
+            return {"username": username}
+
+        # Instrument the app
+        otel_fastapi.FastAPIInstrumentor().instrument_app(
+            app, render_path_parameters=True)
+        client = TestClient(app)
+
+        # Make a request to the endpoint with a path parameter
+        response = client.get("/user/johndoe")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json(), {"username": "johndoe"})
+
+        # Retrieve the spans generated
+        spans = self.memory_exporter.get_finished_spans()
```

Comment:
Are we missing assert the span.name?

Reply:
Assertions have been added. Could you please take another look?